ml-agents/mlagents/trainers/policy/torch_policy.py

```diff
@@ -249,13 +249,13 @@ def increment_step(self, n_steps):
         return self.get_current_step()
 
     def load_weights(self, values: List[np.ndarray]) -> None:
-        pass
+        self.actor_critic.load_state_dict(values)
 
     def init_load_weights(self) -> None:
```

Comment:
We dont need this function

Reply:
Maybe add a comment and mark for removal when TF is deprecated